sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -680,13 +684,14 @@ private case class GpuParquetFileFilterHandler(@transient sqlConf: SQLConf) exte
 
       val (clipped, clippedSchema) =
         withResource(new NvtxRange("clipSchema", NvtxColor.DARK_GREEN)) { _ =>
-          val clippedSchemaTmp = ParquetReadSupport.clipParquetSchema(fileSchema, readDataSchema,
-            isCaseSensitive)
+          val clippedSchemaTmp = ParquetSchemaClipShims.clipSchema(fileSchema, readDataSchema,
+            isCaseSensitive, useFieldId, timestampNTZEnabled)
```

Comment:
Similar "do we need to shim the parameter to a shim?" question here.  Might be simpler to pass the conf to `ParquetSchemaClipShims` and let it handle NTZ internally if that shim needs to do so.

Reply:
Similarly, It can avoid serializing the SQL conf object to executors, and send only the necessary confs to executors.
The two confs are pulled out at driver side, while the `clipSchema` runs on executors.